general/update_fars/01_download_fars.py

```diff
@@ -0,0 +1,15 @@
+from urllib import request
+import os
+
+# place to save data locally
+data_folder = "/mnt/c/Users/tputta/OneDrive - Toole Design/Desktop/SSPF/FARS"
```

Comment:
It probably doesn't matter, but since this is a public repo, do you think that you should put your local filepath in here? Maybe instead make a `data` folder either in the top level repo directory or just in this folder, and save it there?

Reply:
Since it is a one-time thing I didn't bother creating a `data` folder and adding it to gitignore.